devbot/commands/xkcd.py

```diff
@@ -0,0 +1,81 @@
+""" xkcd comic command. """
+import random
+import discord
+from sqlalchemy import or_
+
+import devbot.database as db
+from devbot.registry import Command
+from devbot.tools import api_requests
+
+
+def embed(title, image_url) -> discord.Embed:
+    result = discord.Embed()
+    result.title = title
+    result.set_image(url=image_url)
+    return result
+
+
+@Command(["xkcd"])
+async def xkcd(message_contents, *_args, **_kwargs) -> (str, discord.Embed):
+    """ Returns a xkcd comic """
+    url = "https://xkcd.com/info.0.json"  # latest xkcd
+
+    if message_contents:
+        if len(message_contents) == 1:
```

Comment:
You don't actually have an if statement for the use of `!xkcd` without arguments. As it functions now it wil always return the first ever xkcd. Maybe have it return the latest instead?